doc/options.yaml

```diff
@@ -1049,6 +1067,11 @@ ANKPCUpdateCutoff:
     This option only changes the PC update behavior with the coupled ANK variants.
     Decoupled ANK variants don't get affected by this.
 
+ANKPCUpdateTolAfterCutoff:
+  desc: >
+    The ANK PC update tolerance used after we reach the ``ANKPCUpdateCutoff`` convergence.
```

Comment:
Can you also change the description for `ANKPCUpdateCutoff`? I think the parts about disabling the update should be removed and it should just say that a different tolerance is used.